network-api/networkapi/buyersguide/templates/fragments/article_subnav.html

```diff
@@ -1,37 +1,30 @@
 {% load i18n %}
 
-<div class="article-navbar d-lg-block d-none">
+<div class="article-navbar d-lg-block">
   <div class="container">
     <div class="row">
-      <div class="col px-lg-3 px-0">
-        <div class="d-flex flex-row justify-content-between">
-          <div class="article-navbar-title d-lg-flex d-none pl-0 {% if get_titles %}col-lg-8{% else %}col-lg-12{% endif %}">
-            {% if get_titles %}
-              {{ page.title|truncatechars:60 }}
```

Comment:
Simplified this. Before it was using nested bootstrap stuff, and it wasnt necessary.